Automattic/republication-tracker-tool
# fix: deprecate GA3 code and documentation

### Changes proposed in this Pull Request:



Updates documentation on Google Analytics from UA to GA4. Also adds the following custom event parameters on the `page_view` event which will help with creating reports to analyze only `page_view` events originating from republished pixel fires:

- `shared_post_url`: URL where the post was originally published (duplicate of `page_location` but will only appear on `page_view` events fired by this plugin's tracking pixel)
- `shared_post_id`: WP post ID on the site where it was originally published
- `shared_post_slug`: WP post slug on the site where it was originally published

Also deprecates GA3-related settings and other code related to them.

Closes #189.

### How to test the changes in this Pull Request:

1. Go to Settings > Reading and under the Republication Tracker Tool Settings section, confirm there's no longer a GA3/UA field—just a GA4 ID and GA4 API Secret field.
2. Test the new parameters by setting up the plugin with a GA4 ID + secret on a test site, then copying the post HTML from the widget into a shared post on another test site. View the shared post's front-end on the second site to fire the tracking pixel code.
3. In your GA4 dashboard, go to Reports > Realtime and look for `page_view` events with the custom `shared_post_*` parameters.
4. Review the updated .md docs and screenshots.
